docs/release_notes/release_notes.md

```diff
@@ -12,6 +12,28 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
 For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/master/CHANGELOG.md).
 {% endif %}
 
+## Notable Features
+
+### Puppet 4 Support
+
+Puppet 4 is now supported by Katello.
+
+### Content Host Subscriptions Bulk Actions
+
+Users can now attach Subscriptions to multiple Content Hosts using the UI. 
+
+### Allow adding file type repositories to content views
+
+### Searchable ostree repos
+
+### Create/update Composite Content View by Content View names in hammer
```

Comment:
Versions of published content views can be added or removed from a composite content view via hammer. The version number and content view name can now be used to identify the content view version to add to the composite content view. For example, to add version 1.2 of content view cv1 to composite view composite_cv:

`hammer content-view add-version --name composite_cv --content-view-version 1.2 --content-view-version-content-view-id cv1 --organization 'Default Organization'`

You can remove a version in a similar way:

`hammer content-view remove-version --name composite_cv --content-view-version 1.2 --content-view-version-content-view-id cv1 --organization 'Default Organization'`